packages/compose/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Bug Fixes
+
+-   `useResizeObserver`: fixed a bug that caused elements to not be properly unobserved ([#65389](https://github.com/WordPress/gutenberg/pull/65389)).
+
```

Comment:
If the original `useResizeObserver` is not yet released, then it doesn't need a changelog entry. The package user never had a chance to observe the bugs.

Reply:
True, I thought about this but decided to add the entry anyway, however you're right so I'll remove it!

Reply:
Removed